packages/gatsby-plugin-google-tagmanager/src/gatsby-ssr.js

```diff
@@ -27,7 +41,11 @@ exports.onRenderBody = (
             new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
             j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
             'https://www.googletagmanager.com/gtm.js?id='+i+dl+'${environmentParamStr}';f.parentNode.insertBefore(j,f);
-            })(window,document,'script','dataLayer', '${pluginOptions.id}');`,
+            })(window,document,'script','${
+              pluginOptions.dataLayerName
+                ? pluginOptions.dataLayerName
+                : `dataLayer`
```

Comment:
Maybe shorter? 
```
pluginOptions.dataLayerName || `datalayer`
```

Reply:
Fair enough and I'll make the change.  Thanks for your help.